docs/releasing.md

```diff
@@ -114,10 +114,6 @@ Confirm Lightrider integration will work:
 Now that the integrations are confirmed to work, go back to `lighthouse` folder.
 
 ```sh
-# Run the tests again.
-# Note: you can skip this if you didn't need to land any changes.
-bash ./lighthouse-core/scripts/release/test.sh
```

Comment:
this doesn't really make sense here (if you've just had to land a PR you should be starting over)